src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -646,6 +647,10 @@ def validate_arguments(self):
         if self.get_argument_service_bindings() and len(self.get_argument_service_bindings()) > 1 and self.get_argument_customized_keys():
             raise InvalidArgumentValueError("--bind have multiple values, but --customized-keys only can be set when --bind is single.")
         validate_runtime(self.get_argument_runtime(), self.get_argument_enable_java_metrics(), self.get_argument_enable_java_agent())
+        if self.get_argument_scale_rule_type() and self.get_argument_scale_rule_identity():
```

Comment:
do you have similar validation for update scenario? For example, customer already has scale rule and use update command to specify scale rule identity, will it fail?

Reply:
there is similar validation for update scenario